docs/notebooks/Data.ipynb

```diff
@@ -0,0 +1,1465 @@
+       "\n",
+       "                                       SystemAlertId  ...  \\\n",
+       "0  2518385291989119899_8af9954d-f28d-40ff-a079-d9...  ...   \n",
+       "1               5d60fff6-7dd2-4474-a4d0-4c8e3fa6fad6  ...   \n",
+       "2               b7adb73b-0778-4929-b46a-c0ed642bc61f  ...   \n",
+       "3               11813ab7-ab7c-4719-b0a1-ccb5d4a32223  ...   \n",
+       "4               140b29a6-dc7b-4782-a26a-e52c8de0e1a2  ...   \n",
+       "\n",
+       "                                    RemediationSteps  \\\n",
+       "0  [\\r\\n  \"1. Enforce the use of strong passwords...   \n",
+       "1  [\\r\\n  \"1. Enforce the use of strong passwords...   \n",
+       "2  [\\r\\n  \"1. Review the IP addresses and determi...   \n",
+       "3  [\\r\\n  \"1. Review the IP addresses and determi...   \n",
+       "4  [\\r\\n  \"1. Review the IP addresses and determi...   \n",
+       "\n",
+       "                                  ExtendedProperties  \\\n",
+       "0  {\\r\\n  \"Activity start time (UTC)\": \"2019/07/2...   \n",
+       "1  {\\r\\n  \"Activity start time (UTC)\": \"2019/07/2...   \n",
+       "2  {\\r\\n  \"Destination Port\": \"3389\",\\r\\n  \"Proto...   \n",
+       "3  {\\r\\n  \"Destination Port\": \"3389\",\\r\\n  \"Proto...   \n",
+       "4  {\\r\\n  \"Destination Port\": \"3389\",\\r\\n  \"Proto...   \n",
+       "\n",
+       "                                            Entities SourceSystem  \\\n",
+       "0  [\\r\\n  {\\r\\n    \"$id\": \"4\",\\r\\n    \"HostName\":...    Detection   \n",
+       "1  [\\r\\n  {\\r\\n    \"$id\": \"4\",\\r\\n    \"HostName\":...    Detection   \n",
+       "2  [\\r\\n  {\\r\\n    \"$id\": \"4\",\\r\\n    \"ResourceId...    Detection   \n",
+       "3  [\\r\\n  {\\r\\n    \"$id\": \"4\",\\r\\n    \"ResourceId...    Detection   \n",
+       "4  [\\r\\n  {\\r\\n    \"$id\": \"4\",\\r\\n    \"ResourceId...    Detection   \n",
+       "\n",
+       "                WorkspaceSubscriptionId  WorkspaceResourceGroup  \\\n",
+       "0  3b701f84-d04b-4479-89b1-fa8827eb537e            sentineltest   \n",
+       "1  3b701f84-d04b-4479-89b1-fa8827eb537e            sentineltest   \n",
+       "2                                                                 \n",
+       "3                                                                 \n",
+       "4                                                                 \n",
+       "\n",
+       "                                       ExtendedLinks            ProductName  \\\n",
+       "0  [\\r\\n  {\\r\\n    \"Href\": \"https://interflowwebp...  Azure Security Center   \n",
+       "1  [\\r\\n  {\\r\\n    \"Href\": \"https://interflowwebp...  Azure Security Center   \n",
+       "2  [\\r\\n  {\\r\\n    \"DetailBladeInputs\": \"protecte...  Azure Security Center   \n",
+       "3  [\\r\\n  {\\r\\n    \"DetailBladeInputs\": \"protecte...  Azure Security Center   \n",
+       "4  [\\r\\n  {\\r\\n   
```

Comment:
Maybe we should make a public method to get the query text.

Reply:
Implemented - call query('print') to get a print out of the query